test/tokenbridge.test.ts

```diff
@@ -153,193 +182,229 @@ describe('TokenBridge', async function() {
+    await l1TokenBridge.approveBridge(l1StaticUsdc.address, l2StaticAUsdc.address);
 
     // set L1 token bridge from L2 bridge
-    await l2user.invoke(proxiedTokenBridgeL2, 'initialize_token_bridge', { governor_address: BigInt(l2user.starknetContract.address) });
-    await l2user.invoke(proxiedTokenBridgeL2, 'set_l1_token_bridge', { l1_bridge_address: BigInt(tokenBridgeL1Proxied.address) });
-    const { res: retrievedBridgeAddress } = await proxiedTokenBridgeL2.call('get_l1_token_bridge', {});
-    expect(retrievedBridgeAddress).to.equal(BigInt(tokenBridgeL1Proxied.address));
+    await l2user.invoke(l2TokenBridge, 'initialize_token_bridge', { governor_address: BigInt(l2user.starknetContract.address) });
+    await l2user.invoke(l2TokenBridge, 'set_l1_token_bridge', { l1_bridge_address: BigInt(l1TokenBridge.address) });
+    const { res: retrievedBridgeAddress } = await l2TokenBridge.call('get_l1_token_bridge', {});
+    expect(retrievedBridgeAddress).to.equal(BigInt(l1TokenBridge.address));
 
     // map L1 tokens to L2 tokens on L2 bridge
-    await l2user.invoke(proxiedTokenBridgeL2, 'set_reward_token', { reward_token: BigInt(rewAaveTokenL2.address) });
-    await l2user.invoke(proxiedTokenBridgeL2, 'approve_bridge', { l1_token: BigInt(l1tokenDai.address), l2_token: BigInt(proxiedL2TokenDai.address) });
-    await l2user.invoke(proxiedTokenBridgeL2, 'approve_bridge', { l1_token: BigInt(l1tokenUsdc.address), l2_token: BigInt(proxiedL2TokenUsdc.address) });
+    await l2user.invoke(l2TokenBridge, 'set_reward_token', { reward_token: BigInt(l2rewAAVE.address) });
+    await l2user.invoke(l2TokenBridge, 'approve_bridge', { l1_token: BigInt(l1StaticDai.address), l2_token: BigInt(l2StaticADai.address) });
+    await l2user.invoke(l2TokenBridge, 'approve_bridge', { l1_token: BigInt(l1StaticUsdc.address), l2_token: BigInt(l2StaticAUsdc.address) });
   })
 
-  it('L1 user sends tokens A and tokens B to L2 user', async () => {
-    // approve L1 bridge with max uint256 amount
-    await l1tokenDai.connect(l1user).approve(tokenBridgeL1Proxied.address, MAX_UINT256);
-    await l1tokenUsdc.connect(l1user).approve(tokenBridgeL1Proxied.address, MAX_UINT256);
 
-    // l1user deposits 30 tokens A and 50 tokens B on L1 for l2user on L2
-    l1tokenDaiInitialBalance = await l1tokenDai.balanceOf(l1user.address);
-    l1tokenUsdcInitialBalance = await l1tokenUsdc.balanceOf(l1user.address);
-    txDai = await tokenBridgeL1Proxied.connect(l1user).deposit(l1tokenDai.address, BigInt(l2user.starknetContract.address), 30);
+  it('L1 user bridges staticADai and staticAUsdc to L2 user', async () => {
+    // approve L1 bridge with max uint256 amount
+    await l1StaticDai.connect(l1user).approve(l1TokenBridge.address, MAX_UINT256);
+    await l1StaticUsdc.connect(l1user).approve(l1TokenBridge.address, MAX_UINT256);
+    await dai.connect(l1user).approve(l1TokenBridge.address, MAX_UINT256);
```

Comment:
if I understood correctly, this line and the three following are no longer needed thanks to [this one](https://github.com/aave-starknet-project/aave-starknet-bridge/pull/32/files/766eb086430dafbb50bffd5d9f4a0933b883e3f3..184db62c07c796628019f9dc1b5ae031eb174b81#diff-82d373285757d7293feb043340e157af235cca879cc09a3900b5db6fbda67d18R100)